client/src/components/dashboard/Dashboard.js

```diff
@@ -82,105 +31,38 @@ const Dashboard = () => {
-  const onEditPage = () => {
-    dispatch(addPage(teamId, selectedPage)).then(() => {
-      directToAnotherPage(selectedPage);
-    });
-  };
-
-  const handlePageSelection = (e) => {
-    setSelectedPage(e);
-    setDisplayButton(false);
-  };
-
-  const handleDelete = () => {
-    dispatch(deletePage(teamId, selectedPage));
-    setDeleteModal(false);
-    setSelectedPage(pagePlaceholder);
-  };
-
   return (
     <main>
-      <Modal show={displayAddModal} onHide={closeAddModal} centered size="lg">
-        <Modal.Header closeButton>
-          <Modal.Title id="contained-modal-title-center">
-            Choose the Page you want to add to your website
-          </Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          <DropdownButton
-            className="mx-10"
-            id="dropdown-basic-button"
-            variant="secondary"
-            title={selectedPage}
-            onSelect={handlePageSelection}
-          >
-            {webpageOfferings.map((pageName) => (
-              <Dropdown.Item eventKey={pageName}>{pageName}</Dropdown.Item>
-            ))}
-          </DropdownButton>
-          <Button
-            className="float-right"
-            disabled={displayButton}
-            onClick={onEditPage}
-          >
-            Next
-          </Button>
-        </Modal.Body>
-      </Modal>
-      <Modal
-        show={displayDeleteModal}
-        onHide={closeDeleteModal}
-        centered
-        size="lg"
-      >
-        <Modal.Header closeButton>
-          <Modal.Title id="contained-modal-title-center">
-            Are you sure you want to delete the {selectedPage} page?
-          </Modal.Title>
-        </Modal.Header>
-        <Modal.Body>
-          Deleting the page will NOT remove the data associated with this page,
-          but the page will not be shown on your website
-          <Modal.Footer className="p-0">
-            <Button variant="secondary" onClick={closeDeleteModal}>
-              Cancel
-            </Button>
-            <Button variant="danger" onClick={handleDelete}>
-              Delete This Page
-            </Button>
-          </Modal.Footer>
-        </Modal.Body>
-      </Modal>
       <Container fluid className="p-5">
-        <Card className="text-left" id="table">
-          <Card.Header className="heading1">
-            Web Pages
-            <Button
-              onClick={showAddModal}
-              className="float-right btn btn-primary cardButton buttonPrimary"
-            >
-              Add
-            </Button>
-          </Card.Header>
-          <Card.Body>
-            <Webpages 
-              currentWebPages={currentWebPages} 
-              directToAnotherPage={directToAnotherPage} 
-              showDeleteModal={showDeleteModal}
+        <Tabs
+          defaultActiveKey="home"
+          onSelect={(k) => setCurrentTab(k)}
+          transition={false}
+          className="mb-3"
+        >
+          <Tab eventKey="home" title="Webpages">

```

Comment:
can the eventKey named ''webpages' to make it consistent?

Reply:
At least one of the tabs has to have an eventKey set to 'home' which is the default tab it'll start on when the page loads, if you don't have one set to 'home' there won't be a default tab selected and they'll have to click. See when eventKey is set to webpages:
![image](https://user-images.githubusercontent.com/40584352/129869446-964969ca-6241-40dd-bd91-e672815ca309.png)